<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>liste_mat</class>
 <widget class="QWidget" name="liste_mat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Liste des materiels bureautiques</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font: 10pt;
}
#liste_mat{
	
	background-color: #EEE;
	font: 10pt;
}

.QPushButton{
	border-radius: 2;
	background-color: qlineargradient(spread:pad, x1:0.548, y1:1, x2:0.469455, y2:0, stop:0.627119 rgba(242, 101, 101, 255), stop:1 rgba(255, 146, 117, 255));
	border-bottom: 2px solid;
	border-bottom-color:rgb(221, 83, 83);
	padding: 3px 16px;
	color:rgb(255, 221, 221);	
	font: 10pt;
}
.QPushButton:pressed{
	background-color: rgb(220, 91, 91);
	border-bottom-color: rgb(194, 73, 73);
}

.QLineEdit{
	border-radius: 2;
	border-bottom: 2px solid rgb(166, 166, 166);
	font: 10pt;
	padding: 3px 6px;
	
	background-color:rgb(222, 222, 222);
}

.QComboBox{
	border-radius: 2;
	background-color:rgb(222, 222, 222);
	border: none;
	padding: 3px 6px;
	border-bottom: 2px solid rgb(166, 166, 166);
}
.QComboBox::drop-down{
	 subcontrol-origin: padding;
    subcontrol-position: top right;
	
	border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid;
}
QComboBox:!editable:on, QComboBox::drop-down:editable:on {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #D3D3D3, stop: 0.4 #D8D8D8,
                                stop: 0.5 #DDDDDD, stop: 1.0 #E1E1E1);
}
QHeaderView::section {
	padding-left: 4px;
    border: 1px solid #6c6c6c;
}
QTableView {
	border-bottom: 1px solid #6c6c6c;
}
QTableView:!active {
    selection-background-color: palette(highlight);
    selection-color: palette(highlighted-text);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Etat:</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="annuler_button">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Annuler</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="recherche_combo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Code inventaire</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fournisseur</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Catégorie</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Employé</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="recherche_code">
       <property name="placeholderText">
        <string>Recherche</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="choix_etat">
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>En service</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>En panne</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Stock</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Transferé</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Archive</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QTableWidget" name="tableWidget">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="gridStyle">
      <enum>Qt::SolidLine</enum>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="ajouter_button">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ajouter un nouvel équipement&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="modifier_button">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Modifier les informations de l'équipement selectionné&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Modifier les informations de l'équipement selectionné&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="affecter_button">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Affecter les équipements selectionnés à un employé&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Affecter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="transfer_button">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Transferer vers un autre projet les équipement selectionnés&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Transferer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="qr_button">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Génerer les codes QR pour les équipement seletionnés&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>QR</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>197</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="rechercher_button">
     <property name="text">
      <string>Recherche</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>annuler_button</sender>
   <signal>clicked()</signal>
   <receiver>liste_mat</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>690</x>
     <y>446</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
